provisioning/device/test/_polling_state_machine_test.js

```diff
@@ -196,18 +212,6 @@ describe('polling state machine', function () {
         });
       });
 
-      /* Tests_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_029: [ If `PollingTransport.queryOperationStatus` succeeds with status==Failed, it shall fail with a `DeviceRegistrationFailedError` error ] */
-
-      /* Tests_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_021: [ If `PollingTransport.queryOperationStatus` succeeds with status==Assigned, `register` shall emit an 'operationStatus' event and begin polling for operation status requests. ] */
-      it ('and fires an operationStatus event if it succeeds', function (testCallback) {
```

Comment:
>it [](start = 6, length = 2)

this too

Reply:
Yes.

---
In reply to: [282194210](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282194210) [](ancestors = 282194210)